analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -390,6 +388,8 @@ private const val PACKAGE_SEPARATOR = "# "
  */
 private const val WHY_CHUNK_SIZE = 32
 
+private val JSON: Json by lazy { Json { ignoreUnknownKeys = true } }
```

Comment:
Nit: Creating this is cheap, and it's always required if the GoMod analyzer is used, so I'm unsure whether using `lazy` is beneficial here (also, it was not lazy before).